src/pages/home/report/ReportActionItem.js

```diff
@@ -260,6 +298,20 @@ function ReportActionItem(props) {
                             }
                         />
                     )}
+                    {!props.displayAsGroup && hasBeenFlagged && (
+                        <Button
+                            small
+                            style={[styles.mt2, styles.alignSelfStart]}
+                            onPress={() => setIsHidden(!isHidden)}
+                        >
+                            <Text
+                                style={styles.buttonSmallText}
+                                selectable={false}
+                            >
+                                {isHidden ? 'Reveal message' : 'Hide message'}
```

Comment:
You forgot to translate this which caused a regression https://github.com/Expensify/App/issues/20090

Reply:
Yep, saw that one when it happened... I changed it in one of the places it appears but not the other one. Thanks!